test/TestProjects/sdk/newprojecttypespec/Azure.NewProject.TypeSpec/src/Properties/AssemblyInfo.cs

```diff
@@ -5,7 +5,6 @@
 
 [assembly: InternalsVisibleTo("Azure.NewProject.TypeSpec.Tests, PublicKey = 0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4")]
 
-
```

Comment:
I think we should leave this.  The whitespace is helpful, as this file needs to be hand-edited.

Reply:
Hello @jsquire The build will fail if there is any extra empty line. That's why we need to remove this. Actually there are two empty lines between each section, we just remove the extra empty line.

Reply:
Interesting.  That would imply there is already a single blank line.   The output that I saw had none.   We want to keep a single blank between these.